vertx-spring-boot-samples/vertx-spring-boot-sample-amqp-tls/src/test/java/dev/snowdrop/vertx/sample/amqp/AmqpSampleApplicationTest.java

```diff
@@ -15,9 +17,17 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class AmqpSampleApplicationTest {
 
+    @Autowired
+    ApplicationContext context;
+
     @Autowired
     private WebTestClient client;
 
+    @BeforeEach
+    public void setup() {
+        this.client = WebTestClient.bindToApplicationContext(this.context).build();
```

Comment:
If we are to recreate the client before each step, maybe we shouldn't really use `@Autowired` on it.